clusterctl/examples/google/generate-yaml.sh

```diff
@@ -21,7 +21,13 @@ set -o pipefail
 GCLOUD_PROJECT=$(gcloud config get-value project)
 ZONE=$(gcloud config get-value compute/zone)
 ZONE="${ZONE:-us-central1-f}"
-CLUSTER_NAME=test1
+# Generate a somewhat unique cluster name, UUID is not an option as service-accounts are limited to 30 characters in length
+# and one has a 19 character prefix (i.e. 'machine-controller-'). Of the 11 remaining characters 6 are reserved for the human
```

Comment:
Since names must be so short we should consider making the fixed part of this one much shorter (e.g. `mc-`) at some point. For now I think making the human part be super short is ok.

Reply:
I'll follow up with that change. I agree that 'mc' is fine... for us, but for new hires it will be difficult. Maybe that's just what we have to live with for now.

Reply:
also "cc" for cluster-controller will be a bit cryptic.